loadtest/control/simulcontroller/controller.go

```diff
@@ -192,6 +192,16 @@ func getActionList(c *SimulController) []userAction {
 			run:       control.CreatePersistentNotificationPost,
 			frequency: 0.05,
 		},
+		{
+			name:      "ClickUserProfile",
+			run:       c.clickUserProfile,
+			frequency: 0.03,
```

Comment:
@agarciamontoro - This frequency is calculated assuming 1.0 of `createPost` from SNOW and then multiplying the ratio. But I'll let you handle it since you have been recalculating it.

Reply:
What endpoint did you use to compare against `createPost`?

Reply:
`clickUserProfile` would call out getTeamMember and getChannelMember. I saw that getChannelMember is 10x more frequent than getTeamMember and this is the only common place from where getTeamMember is called. So I selected the getTeamMember's frequency for clickUserProfile, and getChannelMember for clickPermalink. Assuming people click permalinks more often than they open user profiles.

Reply:
Do those actions happen to have a unique telemetry event we can inspect? Relying on endpoints that are used elsewhere should be our last resort, so maybe telemetry can help us here.

Reply:
Nope, just checked. There is no event for "profile picture click". There's something called "api_channels_permalink" but I don't think it's to do with clicking a permalink. So I guess we are on our own here.

Reply:
I see that `getTeamMember` is also called in the team members dropdonw and in some places in the admin console. Assuming we can ignore those, I'm ok with the frequencies as you computed them.

Reply:
Yep, I saw those. I think we can ignore them. Those are like super rare things.